scripts/react_native_pods.rb

```diff
@@ -37,7 +37,7 @@ def use_react_native! (
   path: "../node_modules/react-native",
   fabric_enabled: false,
   new_arch_enabled: ENV['RCT_NEW_ARCH_ENABLED'] == '1',
-  production: false,
+  production: ENV['PRODUCTION'] == false,
```

Comment:
we initially set it `false` like other variables

Reply:
guess this should really be set to `0`
to keep both set and unset value types the same as numbers

Reply:
```suggestion
  production: ENV['PRODUCTION'] == '1',
```
In this case:
* if the `PRODUCTION` parameter is not passed, ENV[`PRODUCTION`] is `nil`, `nil == '1'` evaluates to `false` and then we are NOT in production.
* if we use `PRODUCTION=0`, we have `'0' == '1'` which evaluates to `false` and then we are NOT in production.
* if we use `PRODUCTION=1`, we have `'1' == '1'` which evaluates to `true` and then we are in production.

In your suggestion, the `production` parameter will always be `false`:
* If we don't pass it, `nil == false` evaluates to `false`
* If we pass it with `PRODUCTION=0`, '0 == false` evaluate to `false`
* * If we pass it with `PRODUCTION=1`, '1 == false` evaluate to `false`

Reply:
@cipolleschi yeah just figured 😆💥